src/midi_Defs.h

```diff
@@ -214,7 +214,7 @@ struct RPN
 #define MIDI_CREATE_INSTANCE(Type, SerialPort, Name)                            \
     midi::MidiInterface<Type> Name((Type&)SerialPort);
 
-#if defined(ARDUINO_SAM_DUE) || defined(USBCON)
+#if defined(ARDUINO_SAM_DUE) || defined(USBCON) || defined(__MK20DX128__) || defined(__MK20DX256__) || defined(__MKL26Z64__)
```

Comment:
Are there any other definitions that could be used to precisely identify a Teensy 3.x ?

I'm thinking that if someone else uses the same arch/processor with a different core, they might not end up with the same Serial mapping.

This is highly theoretical though, so if there's no better alternative, I'll merge this change.